daemon/algod/api/server/v2/handlers.go

```diff
@@ -91,6 +105,7 @@ type LedgerForAPI interface {
 	ConsensusParams(r basics.Round) (config.ConsensusParams, error)
 	Latest() basics.Round
 	LookupAsset(rnd basics.Round, addr basics.Address, aidx basics.AssetIndex) (ledgercore.AssetResource, error)
+	LookupAssets(rnd basics.Round, addr basics.Address, assetIDGT basics.AssetIndex, limit uint64) ([]ledgercore.AssetResourceWithIDs, error)
```

Comment:
What is "IDGT"?

Reply:
Asset Id greater than - took from indexer existing pattern.

Reply:
Weird. Because it's actually the start ID, isn't.  That is, it's a value that is sure to be Less Than all the returned values.

I'd prefer something like minAssetID or startAssetID

Reply:
I was trying to be consistent with indexer, (and match what we are putting in the token), but open to adjusting if we see a strong reason to.

Reply:
I just can't imagine anyone possibly guessing what `assetIDGT` means.  That makes it a bad name, even if indexer uses it.  I'd probably rename it in indexer too.

But I admit this is small potatoes. You can decide and resolve this whichever way you go.